templates/talk_registered.md

```diff
@@ -10,7 +10,8 @@ Good luck with your talk!
   **Confirm that you received the meeting key and URL by responding in this issue.**
 - Install a [Zoom client](https://zoom.us/download). This is required for registration-only meetings.
 - Test your setup on your own: check that your sound, video, and slide-sharing work.
-- Announce the meeting to your colleagues: you know best who may be interested.
+- **Announce the meeting to your colleagues** and **invite them to register for the talk**.
```

Comment:
Can boldface the complete sentence as well, so that "and" doesn't feel lonely :)
```suggestion
- **Announce the meeting to your colleagues and invite them to register for the talk**.
```